src/components/ZetkinPerson.tsx

```diff
@@ -15,7 +15,7 @@ const ZetkinPerson: React.FunctionComponent<{
             <Box alignItems="start" display="flex" flexDirection="column" justifyContent="center" ml={ 1 }>
                 <Typography variant="body1">{ name }</Typography>
                 { subtitle && (
-                    <Typography variant="body2">{ subtitle }</Typography>
+                    <span className="MuiTypography-root MuiTypography-body2">{ subtitle }</span>
```

Comment:
What's the purpose of this change?

Reply:
this is bc of the "div can not be a child of p"-thing. If subtitle has the added flair of an icon, it will be inside a Box to make the text look good. and the Box (div) cannot be a child of the Typography (p).

Reply:
I see! Well, a `Box` (`<div>`) should probably also not be inside a `span`, even though there might not be anything complaining about it.

Would it not make sense to render this conditionally, based on whether `subtitle` is complex content (i.e. components) or just text?

What do you think @djbusstop? (No rush, answer tomorrow or Friday).